src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -1574,8 +1574,8 @@ If such a class is used as a base class and if the deriving class defines a dest
   <data name="ERR_BadShiftOperatorSignature" xml:space="preserve">
     <value>The first operand of an overloaded shift operator must have the same type as the containing type, and the type of the second operand must be int</value>
   </data>
-  <data name="ERR_InterfacesCantContainOperators" xml:space="preserve">
-    <value>Interfaces cannot contain operators</value>
+  <data name="ERR_InterfacesCantContainConversionOrEqualityOperators" xml:space="preserve">
+    <value>Interfaces cannot contain conversion, equality, or inequality operators</value>
```

Comment:
I got confused thinking that inequality was <= and related...

---
In reply to: [123673420](https://github.com/dotnet/roslyn/pull/20401#discussion_r123673420) [](ancestors = 123673420)